code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -95,6 +95,11 @@
 	else
 		..()
 
+/obj/item/baton/throw_impact(atom/hit_atom, datum/thrownthing/TT)
+	. = ..()
+	if(ishuman(hit_atom))
```

Comment:
`isliving` would be more appropriate judging by the `apply_hit_effect` args.

Reply:
Also should probably check that `TT` still exists/isn't qdeleted by whatever happens lower in `throw_impact`.